tflite2onnx/op/activation.py

```diff
@@ -68,6 +69,11 @@ def parse(self):
             alpha = self.parseInput(1)
             alpha.shape.insert(0, 1)
 
+        if opcode == tflite.BuiltinOperator.LEAKY_RELU:
+            op_opt = op.BuiltinOptions()
+            option = tflite.LeakyReluOptions()
+            option.Init(op_opt.Bytes, op_opt.Pos)
+            alpha = option.Alpha()
```

Comment:
The alpha needs to be passed to the ONNX operator. The code here actually does nothing. The `Concat` code may tip you how to do it.